node/12/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:12-slim
+FROM node:12-buster-slim
```

Comment:
Cloud you apply this change on the `node:10` image too, plz? It is still used for the PIM 3.2.

Reply:
@damien-carcel , we need to talk :wink:

Reply:
:ah: